files/mygui/openmw_spell_window.layout

```diff
@@ -13,8 +13,16 @@
         <Widget type="SpellView" skin="MW_SpellView" position="8 38 268 490" align="Left Top Stretch" name="SpellView">
         </Widget>
 
-        <!-- Search box-->
-        <Widget type="EditBox" skin="MW_TextBoxEditWithBorder" position="8 535 268 23" align="Left Bottom HStretch" name="FilterEdit">
+        <Widget type="Widget" position="8 535 268 23" align="Left Bottom HStretch">
+            <!-- Spell deletion button -->
+            <Widget type="Button" skin="MW_Button" align="Right Bottom" position="150 0 118 24" name="DeleteSpellButton">
+                <Property key="Caption" value="Delete Spell"/>
```

Comment:
Use GMSTs instead of non localizable strings when you can. sDeleteSpell is applicable for the string value here, so use that instead.